code/modules/mining/equipment/kinetic_crusher/kinetic_crusher.dm

```diff
@@ -168,7 +168,7 @@
 		return
 	var/datum/status_effect/crusher_mark/mark = target.has_status_effect(/datum/status_effect/crusher_mark)
 	var/boosted_mark = mark?.boosted
-	if(world.time < mark.mark_applied + mark.ready_delay) // Simple way to prevent right+left click at the same time to detonate the mark for free
+	if(world.time < mark?.mark_applied + mark?.ready_delay) // Simple way to prevent right+left click at the same time to detonate the mark for free
```

Comment:
None of the code below is going to run if mark doesn't exist, just return if its null before instead of changing these (and change boosted_mark? too)